tuned/plugins/plugin_scheduler.py

```diff
@@ -136,8 +136,230 @@ def get_priority_max(self, sched):
+	to. In other words, child processes created by the `fork()` system
+	call will be processed. Since child processes inherit CPU affinity
+	from their parents, the scheduler plug-in usually does not need to
+	explicitly process these events. As processing perf events can pose
+	a significant CPU overhead, the [option]`perf_process_fork` option
+	parameter is set to `false` by default and child processes are not
+	processed by the scheduler plug-in.
+	+
+	The CPU overhead of the scheduler plugin can be mitigated by using
+	the scheduler [option]`runtime` option and setting it to `0`. This
+	will completely disable the dynamic scheduler functionality and the
+	perf events will not be listened to and acted upon. The disadvantage
+	ot this approach is the procees/thread tuning will be done only at
+	profile application.
+	+
+	.Disabling the scheduler dynamic functionality
+	====
+	----
+	[scheduler]
+	runtime=0
+	isolated_cores=1,3
+	----
+	====
+	+
+	NOTE: For perf events, mmapped buffer is used. Under heavy load
+	the buffer may overflow. In such cases the `scheduler` plug-in
+	may start missing events and failing to process some newly created
+	processes. Increasing the buffer size may help. The buffer size can
+	be set with the [option]`perf_mmap_pages` option. The value of this
+	parameter has to expressed in powers of 2. If it is not the power
+	of 2, the nearest higher power of 2 value is calculated from it
+	and this calculated value used. If the [option]`perf_mmap_pages`
+	option is omitted, the default kernel value is used.
+	+
+	The scheduler plug-in supports process/thread confinement using
+	cgroups v1.
+	+
+	[option]`cgroup_mount_point` option specifies the path to mount the
+	cgroup filesystem or where *TuneD* expects it to be mounted. If unset,
+	`/sys/fs/cgroup/cpuset` is expected.
+	+
+	If [option]`cgroup_groups_init` option is set to `1` *TuneD*
+	will create (and remove) all cgroups defined with the `cgroup*`
+	options.  This is the default behavior. If it is set to `0` the
+	cgroups need to be preset by other means.
+	+
+	If [option]`cgroup_mount_point_init` option is set to `1`,
+	*TuneD* will create (and remove) the cgroup mountpoint. It implies
+	`cgroup_groups_init = 1`. If set to `0` the cgroups mount point
+	needs to be preset by other means. This is the default behavior.
+	+
+	The [option]`cgroup_for_isolated_cores` option is the cgroup
+	name used for the [option]`isolated_cores` option functionality. For
+	example, if a system has 4 CPUs, `isolated_cores=1` means that all
+	processes/threads will be moved to CPUs 0,2-3.
+	The scheduler plug-in will do the isolation by writing the
+	calculated CPU affinity to the `cpuset.cpus` control file of
+	the cgroup specified by the [option]`cgroup_for_isolated_cores`
+	option and moving all the matching processes/threads to this cgroup.
+	If this option is unset, classic
+	cpuset affinity using `sched_setaffinity()` will be used.
+	+
```

Comment:
The scheduler plug-in will isolate the specified core by writing the calculated CPU affinity to the `cpuset.cpus` control file of the specified cgroup and move all the matching processes/threads to this group. If this option is unset, classic cpuset affinity using `sched_setaffinity()` will be used.